cvat-ui/src/components/task-page/details.tsx

```diff
@@ -205,28 +236,28 @@ export default class DetailsComponent extends React.PureComponent<Props, State>
     }
 
     private renderDatasetRepository(): JSX.Element | boolean {
-        const { taskInstance } = this.props;
-        const { repository, repositoryStatus, format } = this.state;
-
+        const { taskInstance, dumpers } = this.props;
+        const {
+            repository, repositoryStatus, format, changedRepo, lfs,
+        } = this.state;
         return (
             !!repository && (
                 <Row>
                     <Col className='cvat-dataset-repository-url'>
                         <Text strong className='cvat-text-color'>
                             Dataset Repository
                         </Text>
+                        <Paragraph editable={{ onChange: this.onChangeRepoValue }}>{changedRepo}</Paragraph>
+                        <Text strong className='cvat-text-color'>Using format</Text>
+                        <Select onChange={this.onChangeFormatValue} className='cvat-repository-format-select' initialValue={format} defaultValue={format}>
```

Comment:
We should do this component managed this way:

```suggestion
                        <Select onChange={this.onChangeFormatValue} className='cvat-repository-format-select' value={format}>
```